src/controller/java/src/matter/controller/model/Paths.kt

```diff
@@ -17,6 +17,8 @@
  */
 package matter.controller.model
 
+import java.util.Optional
```

Comment:
We need to decouple the dependence of java.util.Optional since we have incompatible implementation internally, you might noticed we have put a lot of workaround in Java API to mitigate these issues, for Kotlin, we should not use Java Optional

Reply:
I will change it to `UInt?` .